core/camera/src/test/java/com/google/jetpackcamera/core/camera/test/FakeCameraUseCaseTest.kt

```diff
@@ -59,7 +60,7 @@ class FakeCameraUseCaseTest {
     @Test
     fun canRunCamera() = runTest(testDispatcher) {
         initAndRunCamera()
-        assertThat(cameraUseCase.isPreviewStarted())
+        Truth.assertThat(cameraUseCase.isPreviewStarted())
```

Comment:
Is there any reason add `Truth` at the front?

Reply:
the import changed from importing `Truth.assertThat` to just importing `Truth`.